python/pyspark/mllib/linalg/distributed.py

```diff
@@ -423,7 +423,7 @@ def computeSVD(
         [DenseVector([-0.7071, 0.7071]), DenseVector([-0.7071, -0.7071])]
         >>> svd_model.s
         DenseVector([3.4641, 3.1623])
-        >>> svd_model.V # doctest: +ELLIPSIS
+        >>> svd_model.V # doctest: +SKIP
         DenseMatrix(3, 2, [-0.4082, -0.8165, -0.4082, 0.8944, -0.4472, 0.0], 0)
```

Comment:
Or you can change like this too I believe. 

```suggestion
        DenseMatrix(3, 2, [-0.4082, -0.8165, -0.4082, 0.8944, -0.4472, ...0.0], 0)
```

Reply:
good idea